packages/devtools/lib/src/timeline/frames_bar_plotly.dart

```diff
@@ -0,0 +1,239 @@
+import '../ui/plotly.dart';
+
+class FramesBarPlotly {
+  static const int CPU_GOOD_TRACE = 0;
+  static const int  CPU_JANK_TRACE = 1;
+  static const int  GPU_GOOD_TRACE = 2;
+  static const int  GPU_JANK_TRACE = 3;
+  static const int  CPU_HIGH_WATER_MARK_TRACE = 4;
+  static const int  GPU_HIGH_WATER_MARK_TRACE = 5;
+
+  static const String CPU_COLOR = 'rgba(100, 181, 246, 1)';    // #64B5F6
```

Comment:
Use ThemedColor for all color values referenced in code.
Like the JBColor class in the intellij plugin class it lets us set different colors in light and dark themes if needed.
To convert to css compatible color values you then call
colorToCss(themedColor)
to get a CSS compatible color for a ThemedColor object.
Using the ThemedColor class will also help port our code to Flutter in the future as this Color class is identical to Flutter's.

Reply:
the CPU and GPU colors are in a const in timeline.dart. You can use these instead of duplicating the Color var.